### PR TITLE
[RW-1975][risk=low] rm data browser

### PR DESCRIPTION
**Pull request by calbach**
Removes the bulk of the data browser code, especially anything that currently slows down local dev / Circle.

Still TODO in a follow-up:
- Split out CDR index generation

### CODE REVIEW

**Comment by calbach** [comment]
@freemabd @blrubenstein please review

File: api/libproject/devstart.rb [hunk 1]
```diff
@@ -898,8 +795,8 @@ def generate_local_cdr_db(*args)
 
 Common.register_command({
   :invocation => "generate-local-cdr-db",
-  :description => "generate-cloudsql-cdr --cdr-version <''|YYYYMMDD> --cdr-db-prefix <cdr|public> --bucket <BUCKET>
-Creates and populates local mysql database from data in bucket made by generate-private/public-cdr-counts.",
+  :description => "generate-cloudsql-cdr --cdr-version <''|YYYYMMDD> --cdr-db-prefix <cdr> --bucket <BUCKET>
```
Comment (freemabd) [inline api/libproject/devstart.rb]:
```suggestion
  :description => "generate-cloudsql-cdr --cdr-version <synth_r_20XXqX_X> --cdr-db-prefix <cdr> --bucket <BUCKET>
```

File: api/libproject/devstart.rb [hunk 2]
```diff
@@ -912,7 +809,7 @@ def generate_local_count_dbs(*args)
 Common.register_command({
   :invocation => "generate-local-count-dbs",
   :description => "generate-local-count-dbs.sh --cdr-version <''|YYYYMMDD> --bucket <BUCKET>
```
Comment (freemabd) [inline api/libproject/devstart.rb]:
```suggestion
  :description => "generate-local-count-dbs --cdr-version <synth_r_20XXqX_X> --bucket <BUCKET>
```

File: api/libproject/devstart.rb [hunk 3]
```diff
@@ -841,7 +738,7 @@ def generate_cloudsql_db(cmd_name, *args)
   :invocation => "generate-cloudsql-db",
   :description => "generate-cloudsql-db  --project <PROJECT> --instance <INSTANCE> \
```
Comment (freemabd) [inline api/libproject/devstart.rb]:
```suggestion
  :description => "generate-cloudsql-db  --project <PROJECT> --instance <workbenchmaindb> \
```

File: api/libproject/devstart.rb [hunk 4]
```diff
@@ -841,7 +738,7 @@ def generate_cloudsql_db(cmd_name, *args)
...
 --database <cdrYYYYMMDD> --bucket <BUCKET>
```
Comment (freemabd) [inline api/libproject/devstart.rb]:
```suggestion
--database <synth_r_20XXqX_X> --bucket <BUCKET>
```

**Comment by calbach** [comment]
Merging - will remove the now defunct Synk rule afterwards.

**Pull request merged**